app/Controllers/Items.php

```
@@ -622,6 +623,25 @@ public function getBulkEdit(): string
      */
     public function postSave(int $itemId = NEW_ENTRY): ResponseInterface
```

PHPStan (2.2.7)

[error] Return type (CodeIgniter\HTTP\ResponseInterface) of method App\Controllers\Items::postSave() should be compatible with return type (false) of method App\Controllers\Secure_Controller::postSave()


(method.childReturnType)